app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -53,4 +76,22 @@ class AdministratorControlsActivityPresenter @Inject constructor(
         R.id.administrator_controls_fragment_placeholder
       ) as AdministratorControlsFragment?
   }
+
+  fun loadProfileList() {
+    activity.supportFragmentManager.beginTransaction().add(
+      R.id.administrator_controls_fragment_multipane_placeholder,
+      ProfileListFragment.newInstance(isMultipane)
+    ).commitNow()
+  }
+
+  fun loadAppVersion() {
+    activity.supportFragmentManager.beginTransaction().add(
+      R.id.administrator_controls_fragment_multipane_placeholder,
+      AppVersionFragment()
+    ).commitNow()
+  }
+
+  fun setExtraControlsTitle(title: String) {
+    activity.findViewById<TextView>(R.id.extra_controls_title).text = title
```

Comment:
Change this using data binding / viewmodel

Reply:
Done